agent/ecs_client/model/api/api-2.json

```diff
@@ -731,6 +731,10 @@
         "networkBindings":{"shape":"NetworkBindings"},
         "networkInterfaces":{"shape":"NetworkInterfaces"},
         "healthStatus":{"shape":"HealthStatus"},
+        "managedAgents":{
+	  "shape":"ManagedAgents",
+	  "internalonly":true
```

Comment:
we could remove this field now, else we should keep a note to remove this when its removed.